### ISSUE TITLE
Desktop Push Notifications

### ISSUE THREAD

**Issue by willbarkoff** [PlanHubMe/PlanHub#344]
The user should be able to choose a time, and at that time recieve [Chrome](https://developers.google.com/web/updates/2015/03/push-notifications-on-the-open-web?hl=en)/[Firefox](https://developer.mozilla.org/en-US/docs/Web/API/Push_API)/[Safari](https://developer.apple.com/notifications/safari-push-notifications/) (If one of us ever gets an Apple Developer Program membership) of their assignments due tomorrow. We could also possibly use a [Chrome App](https://developer.chrome.com/apps/notifications) (I am going to convert PlanHub Desktop into a chrome app from an electron app anyway).

**Comment by thatoddmailbox** [comment]
@nywillb Or we could use the [HTML5 Notification API](https://developer.mozilla.org/en-US/docs/Web/API/notification) with [this friendly plugin](https://github.com/alexgibson/notify.js).

**Comment by thatoddmailbox** [comment]
@nywillb Also:

> a chrome app from an electron app anyway

wheye? Chrome apps are still blocked (they have set a rule that even if they are on the whitelist, _apps_ are still blocked)

**Comment by willbarkoff** [comment]
@thatoddmailbox Chrome apps are easier to make

**Comment by thatoddmailbox** [comment]
@nywillb That is true, but Electron apps can do more. Chrome apps should still be capable doing notifications. (of course, so is a regular website with the HTML5 Notification API)

**Comment by amazansky** [comment]
Can't extensions do them or nu? For example if you click popup button HW for next night appears in swal or alert?

**Comment by thatoddmailbox** [comment]
@RobotXLabs Extensions cannot add JavaScript (like swal) to pages without the "Extension would like to access all your data on all websites" permission warning on install. There are browser action buttons that can open a popup window, though.
